File: Java/car.java
class Car{
    private Integer id;
    private String license;
    private Account driver;
    private Integer passengenger;

    //Metodo constructor
    public Car(String license, Account driver) //Dentro del parentesís se colocan los parametros 
    {              //que queremos que sean obligatorios para que se cree un tipo de objeto tipo car.
        this.license = license;
        this.driver = driver;
    }
    //metodo
    void printDataCar(){
        System.out.println("Car License: " + license + "Driver: " + driver.name);
    }

    public Integer getpassengenger()
    {
        return passengenger;
    }

    public void setPassengenger(Integer passengenger)
    {
        this.passengenger = passengenger;
    }
}

File: PHP/car.php
<?php
require_once("account.php");
    class car{
        public $id;
        public $license;
        public $driver;
        public $passangenger;

        public function __construct($name, $document)
        {
            $this->name = $name;
            $this->document = $document;
        }

        public function printDataCar()
        {
            echo "Licencia: $this->license Driver: ".$this->driver->name;
        }
    }
?>
